entities/scripts/seed/bulk-download.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+from database.models import Entity
+from database.models.file import File
+
+from platformics.util.seed_utils import (
+    TEST_USER_ID,
+    SeedSession,
+)
+from support.enums import FileAccessProtocol
+
+
+def main() -> tuple[list[dict[str, str]], dict[str, str]]:
+    """
+    An idempotent seed script to create the minimum viable set of entities to run consensus genomes
+
+    It also creates entity and raw inputs to run the consensus genome workflow
+    """
+    session = SeedSession()
+
+    files = (
+        session.query(File)
+        .join(Entity, File.entity_id == Entity.id)
+        .filter(
+            Entity.owner_user_id == TEST_USER_ID,
+        )
+        .all()
+    )
+
+    # redo above joining File on Entity by
+    file_ids: list[dict[str, str]] = []
+    for file in files:
+        if file.protocol == FileAccessProtocol.s3 and session.s3_local:
+            session.upsert_bucket(file.namespace)
+            print(f"Creating file {file.id} in {file.namespace}/{file.path}")
+            session.s3_local.put_object(Bucket=file.namespace, Key=file.path, Body="ABC")
+        if len(file_ids) < 4:
```

Comment:
Just curious why this is limited on <4?

Reply:
Just thought this made for a nice minimal input for local testing